nuxeo-services/nuxeo-platform-convert/src/test/resources/OSGI-INF/test-bypass-override-contrib.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.platform.convert.tests.bypass.override.converter">
+  <require>org.nuxeo.ecm.platform.convert.tests.bypass.converter</require>
+  <extension target="org.nuxeo.ecm.core.convert.service.ConversionServiceImpl" point="converter">
+
+    <converter name="testBypassFalse" class="org.nuxeo.ecm.platform.convert.tests.LibreOfficeConverterForTest"
+               bypassIfSameMimeType="false">
```

Comment:
`bypassIfSameMimeType` is `false` by default, so it does not override/change the behavior here.

I would have added a converter overriding an existing one (initially with `bypassIfSameMimeType="false"`) with `bypassIfSameMimeType="true"`.

Reply:
Both are present. Since the attribute is initialized to null. I made both senarios. This converter is called testBypassFalse for this purpose (testing the merge) and there is a testBypassTrue as well that is merged with another converter that has null

Reply:
retouched the test so it is relevant and added a default contribution to test default behavior

Reply:
Yep, in fact, what I would do is:
- Change `TestBypassOverride` to `TestBypassIfSameMimeType`
- In this class, have 2 tests:
    - `testContribution` deploying test-bypass-contrib.xml (`@Deploy` at the same level as `@Test`) with 2 contributions:
        - "testBypassFalse" with no bypassIfSameMimeType attribute => default value is false so expect false
        - "testBypassTrue" with bypassIfSameMimeType="true" => value is true so expect true
    - `testContributionOverride` deploying both test-bypass-contrib.xml and test-bypass-override-contrib.xml (2 `@Deploy` at the same level as `@Test`) with test-bypass-override-contrib.xml requiring the previous one and including 2 contributions:
        - "testBypassFalse" with bypassIfSameMimeType="true" => overriding false to true so expect true
        - "testBypassTrue" with bypassIfSameMimeType="false" => overriding true to false so expect false